server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -264,89 +260,163 @@ export class AlfredResourcesFactory implements core.IResourcesFactory<AlfredReso
 			);
 		};
 
+		const tenantThrottlersMap = new Map<string, string>();
+		const tenantGroup1 = config.get("alfred:throttlingGroup:tenantGroup1") as string[];
+		tenantGroup1.forEach((t) => tenantThrottlersMap.set(t, "tenantGroup1"));
```

Comment:
This is a good first pass. I think we should do a follow-up PR to make this more flexible so that we can add more tenantGroups to the configs as necessary, without having to add more code each time. Not 100% necessary on this first PR though.